cpp/unit/producer.cpp

```diff
@@ -209,7 +207,7 @@ void ObjectProducer::initialise(Unit *unit, Player &player) {
 				), 
 				true);
 		}
-		else {
+		else if (this->graphics.count(graphic_type::dying) > 0) {
```

Comment:
you should probably use `.find()` instead of `.count() > 0`... they do the same, but `.count()` can seem a bit... wtf-inducing.